qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -146,28 +154,68 @@ def run(self,
                 DeprecationWarning,
                 stacklevel=3)
 
-        if isinstance(circuits, (QasmQobj, PulseQobj)):
-            warnings.warn('Using a qobj for run() is deprecated and will be '
-                          'removed in a future release.',
-                          PendingDeprecationWarning,
-                          stacklevel=2)
-            qobj = circuits
+        executor = None
+        if backend_options and "executor" in backend_options:
+            executor = backend_options["executor"]
+            del backend_options["executor"]
```

Comment:
Do we really want to support doing this in `backend_options` even though it's deprecated.